File: two-sum/two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let result = []
    for (let i = 0 ; i < nums.length ; i++) {
        for (let b = i+1 ; b < nums.length ; b++) {
            if ((nums[i] + nums[b]) == target) {
                result.push(i)
                result.push(b)
                return result
            }
        }
    }
};

File: two-sum/two-sum.go
package main

import "fmt"

func main() {
	fmt.Println(twoSum([]int{1, 2, 3, 4}, 6))
}

func twoSum(nums []int, target int) []int {

	result := []int{}

	for i := 0; i < len(nums); i++ {
		for b := i + 1; b < len(nums); b++ {
			if (nums[i] + nums[b]) == target {
				result = append(result, i, b)
				return result
			}
		}
	}

	return result
}
